docs/content/guides/rows/rows-sorting.md

```diff
@@ -0,0 +1,3432 @@
+          price: 1890.90,
+          sellDate: '03/05/2023',
+          sellTime: '11:27',
+          inStock: false,
+        },
+        {
+          brand: 'Jetpulse',
+          model: 'Cycling Cap',
+          price: 130.10,
+          sellDate: '27/03/2023',
+          sellTime: '03:17',
+          inStock: true,
+        },
+        {
+          brand: 'Chatterpoint',
+          model: 'Road Tire Tube',
+          price: 59,
+          sellDate: '28/08/2023',
+          sellTime: '08:01',
+          inStock: true,
+        },
+        {
+          brand: 'Chatterpoint',
+          model: 'HL Road Tire',
+          price: 279.99,
+          sellDate: '02/10/2023',
+          sellTime: '13:23',
+          inStock: true,
+        },
+    ]}
+      columns={[
+        {
+          title: 'Brand',
+          type: 'text',
+          data: 'brand',
+        },
+        {
+          title: 'Model',
+          type: 'text',
+          data: 'model',
+        },
+        {
+          title: 'Price',
+          type: 'numeric',
+          data: 'price',
+          numericFormat: {
+            pattern: '$ 0,0.00',
+            culture: 'en-US'
+          },
+          className: 'htLeft',
+        },
+        {
+          title: 'Date',
+          type: 'date',
+          data: 'sellDate',
+          className: 'htRight',
+        },
+        {
+          title: 'Time',
+          type: 'time',
+          data: 'sellTime',
+          className: 'htRight',
+        },
+        {
+          title: 'In stock',
+          type: 'checkbox',
+          data: 'inStock',
+          className: 'htCenter',
+        },
+      ]}
+      multiColumnSorting={{
+        initialConfig: [
+          {
+            // at initialization, sort the data by the 'Brand' column, in ascending order
+            column: 0,
+            sortOrder: 'asc',
+          },
+            // at initialization, sort the data by the 'Model' column, in descending order
+          {
+            column: 1,
+            sortOrder: 'desc',
+          },
+        ],
+      }}
+      licenseKey="non-commercial-and-evaluation"
+    />
+  );
+};
+
+/* start:skip-in-preview */
+ReactDOM.render(<HandsontableComponent />, document.getElementById('example5'));
+/* end:skip-in-preview */
+```
+
+:::
+
+:::
+
+To set an initial sort order, use the [`initialConfig`](@/api/options.md#columnsorting) option. 
+
+::: only-for javascript
+
+```js
+const configurationOptions = {
+  columnSorting: {
+    // at initialization, sort data by the first column, in ascending order
+    initialConfig: {
+      column: 0,
+      sortOrder: 'asc', // for descending order, use `'desc'`
+    },
+};
+```
+
+:::
+
+::: only-for react
+
+```jsx
+<HotTable
+  columnSorting={{
+    // at initialization, sort data by the first column, in ascending order
+    initialConfig: {
+      column: 0,
+      sortOrder: 'asc',  // for descending order, use `'desc'`
+    },
+  }}
+/>
+```
+
+:::
+
+## Add custom sort icons
```

Comment:
WDYT adding a section about custom sort icons for multi-column sorting? Correct me if I'm wrong but right now I can see only a tutorial for single-column sorting.